cli/assets/templates/rust/src/lib.rs

```diff
@@ -1,19 +1,14 @@
+#[cfg(feature = "buddy-alloc")]
+mod alloc;
 mod wasm4;
 use wasm4::*;
 
 const SMILEY: [u8; 8] = [
-    0b11000011,
-    0b10000001,
-    0b00100100,
-    0b00100100,
-    0b00000000,
-    0b00100100,
-    0b10011001,
-    0b11000011,
+    0b11000011, 0b10000001, 0b00100100, 0b00100100, 0b00000000, 0b00100100, 0b10011001, 0b11000011,
```

Comment:
Let's keep the formatting for this as it was, I think it's important for beginners to be able to somewhat visualize the sprite.

Reply:
I'm not so sure about this:

that's the output of [rustfmt](https://github.com/rust-lang/rustfmt):
conforming to language standards, facilitates communication between developers.

Moreover if you develop using an IDE you'll probably get this piece of code formatted properly as soon as you save or commit.
Have you considered using comments to prevent this behavior?

Reply:
Would `#[rustfmt::skip]` work here?